### DISCUSSION TITLE
How to request and track new labels for Frigate+ annotations?

### DISCUSSION THREAD

**Discussion by glossyio** [blakeblackshear/frigate#8532] [Ask A Question] [answered]
I would like to request a new label for the annotations on Frigate+ custom models. What is the current process to request a new label, track progress towards it, and/or prioritize the new feature?

My apologies for asking this. I know you have addressed this in previous issues and discussions, but I wanted to know if there was a pinned down methodology for it. 

Specifically, I would like to label *bicycles* in Frigate+ to better track bike counts over time and compare and contrast to car and pedestrian counts. The base model is abysmal at recognizing, categorizing, and differentiating bikes from other object types on my system. Not to mention cargo bikes and ebikes not being counted--recognizing and differentiating these would also be of value for me.

Thank you!

**Comment by NickM-27** [comment, marked as answer]
I believe bicycle is already on the potential list. In general I think commenting on https://github.com/blakeblackshear/frigate/discussions/7932 is best.

**Reply by glossyio** [reply]
Thank you! I'll add a comment to that thread. Still very excited about Frigate+ and being able to use it for this purpose.
